contracts/nns/contract.go

```diff
@@ -613,6 +650,24 @@ func DeleteRecords(name string, typ recordtype.Type) {
 	updateSoaSerial(ctx, tokenID)
 }
 
+// DeleteNeoRecord removes domain record of the Neo type for the specified address in
+// 20-byte hash format. The name MUST NOT be a TLD.
+func DeleteNeoRecord(name string, address interop.Hash160) {
+	ctx := storage.GetContext()
+	tokenID := []byte(tokenIDFromName(ctx, name))
+
+	fragments := std.StringSplit(string(tokenID), ".")
+	if len(fragments) == 1 {
+		panic("token not found")
+	}
+
+	ns := getFragmentedNameState(ctx, tokenID, fragments)
+	ns.checkAdmin()
+	key := getNeoRecordKey(tokenID, name, address)
```

Comment:
Here `address` is still not checked and I can trigger `storage.Delete` with zero (or any other) length `address`